src/app/inventory/ConnectedInventoryItem.tsx

```diff
@@ -43,8 +44,8 @@ export default function ConnectedInventoryItem({
   const searchHidden =
     // dim this item if there's no search filter and it's archived
     (dimArchived && defaultFilterActive && tag === 'archive') ||
-    // or if there is filtering and it doesn't meet the condition
-    (allowFilter && !currentFilter(item));
+    // or if there is a valid filter and it doesn't meet the condition
+    (allowFilter && !validQuery && !currentFilter(item));
```

Comment:
so confused by the ! on validQuery, what am i missing

Reply:
or if the plan is to dim everything when there's an invalid filter, shouldn't it not be ||'d with the other stuff?

Reply:
@bhollis wanted no items greyed out if the search is invalid.

Reply:
if this is determining whether it should be dimmed (searchHidden) then, and all things should not be when there's an invalid query, why isn't this dimmed = etc etc etc &&validquery etc etc

`if there is a valid filter` doesn't seem to match `!validQuery` still confused sorry

Reply:
Oops yes, mistake on my part, thanks. I wonder if this should still have the default behavior of dimming the archived items or not.